pkg/ddevapp/config.go

```diff
@@ -224,7 +224,7 @@ func (c *Config) RenderComposeYAML() (string, error) {
 		// path.Join is desired over filepath.Join here,
 		// as we always want a unix-style path for the mount.
 		"docroot":     path.Join("../", c.Docroot),
-		"plugin":      c.Platform,
+		"plugin":      "ddev",
```

Comment:
The platform is "local", but we want the names in the template to be "ddev". This is also solvable by changing the name of the template variable from "plugin" to something more appropriate. Better yet, I will look at just changing the template itself.